lib/dialects/oracledb/index.js

```diff
@@ -259,100 +258,100 @@ Client_Oracledb.prototype._query = function(connection, obj) {
+        const ret = obj.outBinding[line];
 
-      offset =
-        offset +
-        (obj.outBinding[line - 1] ? obj.outBinding[line - 1].length : 0);
+        offset =
+          offset +
+          (obj.outBinding[line - 1] ? obj.outBinding[line - 1].length : 0);
 
-      for (let index = 0; index < ret.length; index++) {
-        const out = ret[index];
+        for (let index = 0; index < ret.length; index++) {
+          const out = ret[index];
 
-        await new Promise(function(bindResolver, bindRejecter) {
-          if (out instanceof BlobHelper) {
-            const blob = outBinds[index + offset];
-            if (out.returning) {
+          await new Promise(function(bindResolver, bindRejecter) {
+            if (out instanceof BlobHelper) {
+              const blob = outBinds[index + offset];
+              if (out.returning) {
+                obj.response[line] = obj.response[line] || {};
+                obj.response[line][out.columnName] = out.value;
+              }
+              blob.on('error', function(err) {
+                bindRejecter(err);
+              });
+              blob.on('finish', function() {
+                bindResolver();
+              });
+              blob.write(out.value);
+              blob.end();
+            } else if (obj.outBinding[line][index] === 'ROWID') {
+              rowIds.push(outBinds[index + offset]);
+              bindResolver();
+            } else {
               obj.response[line] = obj.response[line] || {};
-              obj.response[line][out.columnName] = out.value;
-            }
-            blob.on('error', function(err) {
-              bindRejecter(err);
-            });
-            blob.on('finish', function() {
+              obj.response[line][out] = outBinds[index + offset];
               bindResolver();
-            });
-            blob.write(out.value);
-            blob.end();
-          } else if (obj.outBinding[line][index] === 'ROWID') {
-            rowIds.push(outBinds[index + offset]);
-            bindResolver();
-          } else {
-            obj.response[line] = obj.response[line] || {};
-            obj.response[line][out] = outBinds[index + offset];
-            bindResolver();
-          }
-        });
+            }
+          });
+        }
+      }
+      if (connection.isTransaction) {
+        return obj;
+      }
+      await connection.commitAsync();
+      if (obj.returningSql) {
+        const response = await connection.executeAsync(
+          obj.returningSql(),
+          rowIds,
+          { resultSet: true }
+        );
+        obj.response = response.rows;
       }
-    }
-    if (connection.isTransaction) {
       return obj;
-    }
-    await connection.commitAsync();
-    if (obj.returningSql) {
-      const response = await connection.executeAsync(
-        obj.returningSql(),
-        rowIds,
-        { resultSet: true }
-      );
-      obj.response = response.rows;
-    }
-    return obj;
-  });
```

Comment:
This is literally all just a gigantic re-indentation.  (I confirmed this by copying the content from both branches, normalizing the indentation, and comparing the diffs.  End-result: the diffs were identical)